Antidote/ActiveCallViewController.m

```diff
@@ -319,6 +339,84 @@ - (void)showCallPausedByFriend
     [self createFriendAvatar];
 }
 
+- (void)setVideoView:(UIView *)videoView
+{
+
+    if (_videoView == videoView) {
+        return;
+    }
+
+    if (_videoView) {
+        [_videoView removeFromSuperview];
+    }
+
+    _videoView = videoView;
+
+    if (! videoView) {
+
+        if (! self.previewLayer) {
+            [self updateCallsControlViewExpand];
+        }
+
+        return;
+    }
+
+
+    if (self.previewView) {
+        [self.view insertSubview:self.videoView belowSubview:self.previewView];
+    }
+    else {
+        [self.view addSubview:self.videoView];
+    }
+
+    [self.videoView makeConstraints:^(MASConstraintMaker *make) {
+        make.bottom.equalTo(self.endCallButton.top).with.offset(-kIndent);
+        make.top.equalTo(self.topViewContainer.bottom).with.offset(kIndent);
+        make.left.equalTo(self.view);
+        make.right.equalTo(self.view);
+    }];
```

Comment:
Will `videoView` be resized properly? Or it will have wrong proportions (fix for that can go to next pull requests)